RELEASE-NOTES.txt

```diff
@@ -2,6 +2,7 @@
 
 17.1
 -----
+- [*] Blaze: Stop showing Blaze in dashboard when site not eligible. [https://github.com/woocommerce/woocommerce-ios/pull/11797]
```

Comment:
Nit: Maybe we can reword this to make it clearer?

```suggestion
- [*] Blaze: Hide the Blaze section on the Dashboard screen when logged in without WPCom. [https://github.com/woocommerce/woocommerce-ios/pull/11797]
```

Reply:
I agree. Done in https://github.com/woocommerce/woocommerce-ios/pull/11797/commits/c3269c29b010e42fe4404972c6080bc554e6d5af